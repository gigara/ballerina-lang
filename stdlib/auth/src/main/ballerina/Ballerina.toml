[project]
org-name = "ballerina"
version = "0.0.0"

[platform]
target = "java"

    [[platform.libraries]]
    artafactId = "auth"
    version = "@project.version@"
    path = "./lib/ballerina-auth-@project.version@.jar"
    groupId = "ballerina"
    modules = ["auth"]

    [[platform.libraries]]
    artafactId = "commons-logging"
    version = "1.1.1"
    path = "./lib/commons-logging-1.1.1.jar"
    groupId = "commons-logging"
    modules = ["auth"]
