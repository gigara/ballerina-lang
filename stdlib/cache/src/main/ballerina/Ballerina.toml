[project]
org-name = "ballerina"
version = "0.0.0"

[platform]
target = "java"

    [[platform.libraries]]
    artafactId = "commons-logging"
    version = "1.1.1"
    path = "./lib/commons-logging-1.1.1.jar"
    groupId = "commons-logging"
    modules = ["cache"]
