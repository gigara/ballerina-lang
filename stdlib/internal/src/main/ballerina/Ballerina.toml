[project]
org-name = "ballerina"
version = "0.0.0"

[platform]
target = "java"

    [[platform.libraries]]
    artafactId = "internal"
    version = "@project.version@"
    path = "./lib/ballerina-internal-@project.version@.jar"
    groupId = "ballerina"
    modules = ["internal"]
