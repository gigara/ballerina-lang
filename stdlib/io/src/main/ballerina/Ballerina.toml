[project]
org-name = "ballerina"
version = "0.0.0"

[platform]
target = "java"

    [[platform.libraries]]
    artafactId = "io"
    version = "@project.version@"
    path = "./lib/ballerina-io-@project.version@.jar"
    groupId = "ballerina"
    modules = ["io"]
