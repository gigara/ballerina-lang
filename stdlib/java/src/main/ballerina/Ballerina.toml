[project]
org-name = "ballerinax"
version = "0.0.0"

[platform]
target = "java"

    [[platform.libraries]]
    artafactId = "java"
    version = "@project.version@"
    path = "./lib/ballerina-java-@project.version@.jar"
    groupId = "ballerina"
    modules = ["java"]