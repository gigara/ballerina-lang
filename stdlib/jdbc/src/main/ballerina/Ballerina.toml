[project]
org-name = "ballerinax"
version = "0.0.0"

[platform]
target = "java"

    [[platform.libraries]]
    artafactId = "jdbc"
    version = "@project.version@"
    path = "./lib/ballerina-jdbc-@project.version@.jar"
    groupId = "ballerina"
    modules = ["jdbc"]
