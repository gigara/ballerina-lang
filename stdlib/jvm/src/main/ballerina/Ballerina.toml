[project]
org-name = "ballerina"
version = "0.0.0"

[platform]
target = "java"

    [[platform.libraries]]
    artafactId = "jvm"
    version = "@project.version@"
    path = "./lib/ballerina-jvm-@project.version@.jar"
    groupId = "ballerina"
    modules = ["jvm"]

    [[platform.libraries]]
    artafactId = "asm"
    version = "6.2.1"
    path = "./lib/asm-6.2.1.jar"
    groupId = "org.ow2.asm"
    modules = ["jvm"]
