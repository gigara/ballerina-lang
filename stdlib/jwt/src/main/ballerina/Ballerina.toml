[project]
org-name = "ballerina"
version = "0.0.0"

[platform]
target = "java"

    [[platform.libraries]]
    artafactId = "jwt"
    version = "@project.version@"
    path = "./lib/ballerina-jwt-@project.version@.jar"
    groupId = "ballerina"
    modules = ["jwt"]
