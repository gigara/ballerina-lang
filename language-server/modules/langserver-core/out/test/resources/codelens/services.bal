import ballerina/http;
import ballerina/task;
import ballerina/io;
import ballerina/websub;
import ballerina/log;

service httpService on new http:Listener(9090) {
    resource function sayHello(http:Caller caller, http:Request req) {
    }
}

listener http:Listener securedListener = new(
                                             9092,
                                             config = {secureSocket: {
                                                 keyStore: {
                                                     path: "${ballerina.home}/bre/security/ballerinaKeystore.p12",
                                                     password: "ballerina"
                                                 },
                                                 trustStore: {
                                                     path: "${ballerina.home}/bre/security/ballerinaTruststore.p12",
                                                     password: "ballerina"
                                                 }
                                             }
                                             }
);

service httpsService on securedListener {
    resource function sayHello(http:Caller caller, http:Request req) {
    }
}

service wssService on new http:WebSocketListener(9094) {
    resource function onOpen(http:WebSocketCaller caller) {
    }
    resource function onText(http:WebSocketCaller caller, string text, boolean finalFrame) {
        io:println("received: " + text);
    }
    resource function onClose(http:WebSocketCaller caller, int statusCode, string reason) {
    }
}

listener http:WebSocketListener wsEP = new(
                                           9093,
                                           config = {secureSocket: {
                                               keyStore: {
                                                   path: "${ballerina.home}/bre/security/ballerinaKeystore.p12",
                                                   password: "ballerina"
                                               },
                                               trustStore: {
                                                   path: "${ballerina.home}/bre/security/ballerinaTruststore.p12",
                                                   password: "ballerina"
                                               }
                                           }
                                           }
);

service wsssService on wsEP {
    resource function onOpen(http:WebSocketCaller caller) {
    }
    resource function onText(http:WebSocketCaller caller, string text, boolean finalFrame) {
        io:println("received: " + text);
    }
    resource function onClose(http:WebSocketCaller caller, int statusCode, string reason) {
    }
}

service websubSubscriber on websubEP {
    resource function onIntentVerification(websub:Caller caller, websub:IntentVerificationRequest request) {
    }
    resource function onNotification(websub:Notification notification) {
    }
}

listener websub:Listener websubEP = new websub:Listener(8181);