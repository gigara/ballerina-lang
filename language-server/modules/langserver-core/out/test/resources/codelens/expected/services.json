{
    "result": [
        {
            "range": {
                "start": {
                    "line": 6,
                    "character": 1
                },
                "end": {
                    "line": 6,
                    "character": 1
                }
            },
            "command": {
                "title": "Show API Design",
                "command": "ballerina.showAPIEditor",
                "arguments": [
                    {
                        "argumentK": "service.name",
                        "argumentV": "httpService"
                    }
                ]
            }
        },
        {
            "range": {
                "start": {
                    "line": 26,
                    "character": 1
                },
                "end": {
                    "line": 26,
                    "character": 1
                }
            },
            "command": {
                "title": "Show API Design",
                "command": "ballerina.showAPIEditor",
                "arguments": [
                    {
                        "argumentK": "service.name",
                        "argumentV": "httpsService"
                    }
                ]
            }
        }
    ],
    "id": {
        "left": "324"
    },
    "jsonrpc": "2.0"
}